picoquic/pictls_openssl_crypto.c

```diff
@@ -0,0 +1,25 @@
+/*
+* Copyright (c) 2020, The PicoQuic Authors.
+* All rights reserved.
```

Comment:
I am OK with copyright to a specific author, but I also need a copyright to Private Octopus Inc., maybe delegated. Otherwise, source management could become very messy.

Reply:
Also, please check the name. Cannot be "pictls_". Should be either "ptls_", the convention in picotls, or "picoquic". Or maybe just "openssl_crypto.c".

Reply:
True, I thought we could change it to a more generic Copyright since I am not part of Private Octopus Inc., I dont care to much about copyright, what about:

```c
/*
* Copyright (c) 2020, Private Octopus Inc., The PicoQuic Authors.
* All rights reserved.
```

This was inspired by https://github.com/openthread/openthread/blob/master/LICENSE.

Reply:
Private Octopus is my one person company. Not that I am making any money with the code, but i would rather have a clean copyright there. At some point I could set up something like a non-profit picoquic foundation, but that requires a fair bit of lawyers fee, so I would only do it if there is actual demand.

What you propose here, " Copyright (c) 2020, Private Octopus Inc., The PicoQuic Authors",  is fine.